### PR TITLE
Aggregator favicon, spacing, open color and text clamp

### PR DESCRIPTION
**Pull request by peterpeterparker**
# Changes

- replace empty favicon with IC favicon
- improve article spacing and alignment (row-gap, align-self etc.)
- clamp Snses titles on three lines
- improve render performance (`append(...)` instead of x `appendChild`)
- new background color for status "open"

# Screenshots

Capture d’écran 2023-08-21 à 07 22 24

### CODE REVIEW

File: CHANGELOG-Sns_Aggregator.md [hunk 1]
```diff
@@ -18,9 +36,8 @@ The SNS Aggregator is released through proposals in the Network Nervous System.
 ### Fixed
 
 - Update the index.html when it has changed.
-
-* More tests that the SNS aggregator contains the expected number of SNSs.
-* Display commit, branch name and similar data when deploying to a test canister.
+- More tests that the SNS aggregator contains the expected number of SNSs.
```
Comment (mstrasinskis) [inline CHANGELOG-Sns_Aggregator.md:39]: Is this from another PR?

File: CHANGELOG-Sns_Aggregator.md [hunk 2]
```diff
@@ -10,6 +10,24 @@ The SNS Aggregator is released through proposals in the Network Nervous System.
...
+
+### Security
+
+### Not Published
+
+### Removed
+
+### Deprecated
+
+## [Proposal 124250](https://nns.ic0.app/proposal/?u=qoctq-giaaa-aaaaa-aaaea-cai&proposal=124250)
```
Comment (mstrasinskis) [inline CHANGELOG-Sns_Aggregator.md:27]: This proposal could be rejected because the nns-dapp proposal was due to the hot-fix last Friday.

**Review by mstrasinskis — COMMENTED** [review]
@peterpeterparker Let's wait a bit. Not sure about the proposal number.

File: CHANGELOG-Sns_Aggregator.md (hunk 2 above)
Comment (peterpeterparker) [reply CHANGELOG-Sns_Aggregator.md]: Removed.

File: CHANGELOG-Sns_Aggregator.md (hunk 1 above)
Comment (peterpeterparker) [reply CHANGELOG-Sns_Aggregator.md]: It's fixed now.

File: CHANGELOG-Sns_Aggregator.md (hunk 2 above)
Comment (mstrasinskis) [reply CHANGELOG-Sns_Aggregator.md]: Sorry for being too safe. All is good, the proposal was executed 👍

**Review by mstrasinskis — APPROVED** [review]
Thank you, nice improvements!

**Pull request merged**